[2024-03-21 00:53:02,143] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.retrieve_raw_data 2024-03-21T00:52:00+00:00 [queued]>
[2024-03-21 00:53:02,164] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.retrieve_raw_data 2024-03-21T00:52:00+00:00 [queued]>
[2024-03-21 00:53:02,164] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-21 00:53:02,164] {taskinstance.py:1088} INFO - Starting attempt 1 of 6
[2024-03-21 00:53:02,164] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-21 00:53:02,188] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): retrieve_raw_data> on 2024-03-21T00:52:00+00:00
[2024-03-21 00:53:02,199] {standard_task_runner.py:52} INFO - Started process 6993 to run task
[2024-03-21 00:53:02,204] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'retrieve_raw_data', '2024-03-21T00:52:00+00:00', '--job-id', '8596', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpruvcn9nv', '--error-file', '/tmp/tmp3knmhp2l']
[2024-03-21 00:53:02,204] {standard_task_runner.py:77} INFO - Job 8596: Subtask retrieve_raw_data
[2024-03-21 00:53:02,271] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.retrieve_raw_data 2024-03-21T00:52:00+00:00 [running]> on host 3a3d0401ac26
[2024-03-21 00:53:02,369] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=retrieve_raw_data
AIRFLOW_CTX_EXECUTION_DATE=2024-03-21T00:52:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-21T00:52:00+00:00
[2024-03-21 00:53:02,818] {python.py:151} INFO - Done. Returned value was: None
[2024-03-21 00:53:02,832] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=retrieve_raw_data, execution_date=20240321T005200, start_date=20240321T005302, end_date=20240321T005302
[2024-03-21 00:53:02,877] {taskinstance.py:1265} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-03-21 00:53:02,899] {local_task_job.py:151} INFO - Task exited with return code 0
